pkg/agent/loadbalancer/loadbalancer_test.go

```diff
@@ -105,7 +105,7 @@ func TestFailOver(t *testing.T) {
 		DataDir:   tmpDir,
 	}
 
-	lb, err := New(cfg.DataDir, SupervisorServiceName, cfg.ServerURL)
+	lb, err := New(cfg.DataDir, SupervisorServiceName, cfg.ServerURL, 0)
```

Comment:
Can we make a const for the `0` value and name it `defaultPort` or something similarly descriptive?

Reply:
Do we need consts in tests? I feel like it's generally OK to have weird literals scattered about in tests since you're feeding it specific variables as part of the setup.